docs/firefighting/index.md

```diff
@@ -121,9 +121,9 @@ $ cchq <env> service commcare stop
 
 ## Couch node is down
 
-If a couch node is down, it could mean that it is either very slow at responding to requests or it has stopped running.
+If a couch node is down, the couch disk might be full. In that case, see "Couch node data disk is full" below. Otherwise, it could mean that the node is slow to respond, erroring frequently, or that the couch process or VM itself in a stopped state.
```

Comment:
```suggestion
If a couch node is down, the couch disk might be full. In that case, see [Couch node data disk is full](#couch-node-data-disk-is-full) below. Otherwise, it could mean that the node is slow to respond, erroring frequently, or that the couch process or VM itself in a stopped state.
```